ktor-server/ktor-server-plugins/ktor-server-sessions/jvm/src/io/ktor/server/sessions/KotlinxSessionSerializer.kt

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.sessions
+
+import io.ktor.http.*
+import kotlinx.serialization.*
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.encoding.*
+import kotlinx.serialization.internal.*
+import kotlinx.serialization.modules.*
+
+/**
+ * Returns a [SessionSerializer] based on `kotlinx.serialization` library.
+ */
+public inline fun <reified T : Any> kotlinxSessionSerializer(
```

Comment:
Could we make it scoped? I would avoid adding session specific top-level symbols

Reply:
it looks like the constructor function `KotlinxSessionSerializer`

Reply:
Btw I can't find any public usages of it